### ISSUE TITLE
Opt-out permission for posting event pictures to Instagram

### ISSUE THREAD

**Issue by CatoOri** [svthalia/concrexit#2592]
**Problem** 
We (Paparazcie) want the Thalia Instagram to reflect the vibe in Thalia more, and this is really hard without pictures of events. It is very frustrating to contact everyone in event pictures to get permission to post event pictures that they are on. In our experience, people don't object to posting the pictures, but are slow to respond. 

**Possible solution** 
It would be nice to have a checkbox in your Thalia profile to opt-out of consent for posting pictures with you in it, and to have this choice be visible in the attendance list of an event. This way, Paparazcie can independently check if people consent to their picture being posted without having to approach everyone in the picture per picture. 

**Alternative solution** 
We considered having this checkbox when registering for an event, but this seems like more hassle.

**Comment by se-bastiaan** [comment]
Have you considered not asking for permission? Because you don't have to. Thalia's privacy policy says the following:

>Thalia uses photo’s which are made during her events for the promotion of events and the association. These photo's can be made available on the website of Thalia for members or they can be posted on social media accounts of Thalia. If such image is posted on a social media platform other than the website of Thalia, permission will be asked to all recognisable persons in the picture if this is feasible. It is possible to ask for the removal of a certain photo from social media when someone on that photo desires so, by sending a mail to [info@thalia.nu](mailto:info@thalia.nu). Processing of this data happens on the basis of it being a legitimate interest of Thalia. When someone wishes not to be photographed, they can indicate this to the photographer. A request of deletion of a certain photo can also be made after it has been taken, by sending a mail to [info@thalia.nu](mailto:info@thalia.nu).

**Comment by se-bastiaan** [comment]
We cannot easily add this information to the profile, that would require us to give access to the full object. We could create a blacklist in the photos application however.

Do note that this _cannot_ be an opt-out due to the way the privacy policy is worded. People have to be asked for permission for each photo technically.

**Comment by CatoOri** [comment]
But this quite literary states that we will ask permission for each individual picture, not asking is not possible then. 

What would the blacklist entail? Would it be about persons or about specific pictures?

**Comment by se-bastiaan** [comment]
Yes I think we added that in the latest revision 😅  Adjusted my comment. My bad.

With a blacklist I would mean a list of people that don't want to be on a photo. Or since it should be opt-in it would more likely be a white/allowlist.

**Comment by CatoOri** [comment]
I would prefer an opt-out version, since I think it will a lot of effort to move people to opt-in. But if it is the only feasible option, any white-list would be nice to have :)

**Comment by se-bastiaan** [comment]
Yes but opt-out is really no option due to the way the privacy policy dictates that people will have to be asked. It would mean 'we will not ask, if you don't want it you will have to let us know'. It has to be 'we will not use photos unless you specifically give us permission'

**Comment by DeD1rk** [comment]
We can ask people when registering for thalia, and bump with a banner for current members until people either allow/deny
